### PR TITLE
hiroki changed the story  

### PR DESCRIPTION
**Pull request by hirokisaru**
here what i did .............. bra bra bra 2020 01 10

### CODE REVIEW

File: story.txt [hunk 1]
```diff
@@ -4,3 +4,5 @@ Hiroki asked me a question about how to open previous changes.
 
 I completely ignored Hiroki, and decided to buy a fish
 a crsipy fish to slap Ashbin in the hamstrangs.
+
+// Hiroki changed 2021 01 09
```
Comment (ashby-git) [inline story.txt:8]: This is just empty lines. Please add content to the story.

**Review by ashby-git — CHANGES_REQUESTED** [review]

File: story.txt (hunk 1 above)
Comment (hirokisaru) [reply story.txt:8]: @ashby-git fixed 2020 01 10

Comment (ashby-git) [reply story.txt:8]: :(

**Review by ashby-git — APPROVED** [review]
:(

**Pull request merged**